network/p2p/middleware.go

```diff
@@ -70,8 +81,6 @@ func NewMiddleware(log zerolog.Logger,
 	libP2PNodeFactory LibP2PFactoryFunc,
 	flowID flow.Identifier,
 	metrics module.NetworkMetrics,
-	maxUnicastMsgSize int,
```

Comment:
just using the constants defined in the file instead of passing these two params in.